corehq/apps/geospatial/static/geospatial/js/models.js

```diff
@@ -695,6 +695,15 @@ hqDefine('geospatial/js/models', [
         };
 
         self.deleteSelectedPolygonFilter = function () {
+            if (self.mapObj.hasDisbursementLayers()) {
+                if (confirmForClearingDisbursement()) {
+                    self.mapObj.removeDisbursementLayers();
+                    $('#disbursement-clear-message').show();
```

Comment:
@ajeety4 
Nit: since it's also used [here](https://github.com/dimagi/commcare-hq/pull/35257/commits/c765571b3fae56bf290e1cf59bb241a364642da4) I think it might be helpful to wrap these 2 lines in a function called something like "clearDisbursementResults" (or similar).

Reply:
Then it will read nicely
```
if (confirmForClearingDisbursement()) {
    clearDisbursementResults();
}
```

Reply:
Actually, ignore the above. I see you did dry out the code by introducing `clearDisbursementBeforeProceeding`.